chp08_objects/example_08_02_twoobjects/sketch.js

```diff
@@ -21,25 +21,27 @@ function draw() {
   car2.display();
 }
 
-// Even though there are multiple objects, we still only need one constructor function. 
-// No matter how many cookies we make, only one cookie cutter is needed.Isn’t object-oriented programming swell?
-function Car(tempC, tempXpos, tempYpos, tempXspeed) { // The Constructor is defined with arguments.
-  this.c = tempC;
-  this.xpos = tempXpos;
-  this.ypos = tempYpos;
-  this.xspeed = tempXspeed;
+// Even though there are multiple objects, we still only need one constructor function.
+// No matter how many cookies we make, only one cookie cutter is needed. Isn’t object-oriented programming swell?
+class Car {
+    constructor(tempC, tempXpos, tempYpos, tempXspeed) { // The Constructor is defined with arguments.
```

Comment:
I can't decide about this, but in JS I'm now just naming the constructor parameters with the same variable name, i.e. `xpos` rather than the "temp" prefix `tempXpos`. This is a direct port so maybe ok to leave the same, what do you think?

Reply:
I copied them exactly here, but noticed later (chapter 17 on) that constructor arguments did not include the `temp` prefix. I think this is a good progression to show how constructor arguments work, and then *later* show that they can have the same name as fields, using or omitting `this.` to specify which we mean, as this may be confusing when first being introduced to classes. I am happy to change them over though, let me know your thoughts.